gix-blame/src/file/function.rs

```diff
@@ -400,12 +424,19 @@ fn find_path_entry_in_commit(
     file_path: &BStr,
     buf: &mut Vec<u8>,
     buf2: &mut Vec<u8>,
+    stats: &mut Statistics,
 ) -> Option<gix_object::tree::Entry> {
     let commit_id = odb.find_commit(commit, buf).unwrap().tree();
     let tree_iter = odb.find_tree_iter(&commit_id, buf).unwrap();
+    stats.commits_to_tree += 1;
```

Comment:
Minor issue: should this line come before `find_tree_iter` as `unwrap` might fail in which case `+= 1` would not run?

Reply:
You are right. Generally I didn't care too much about the position as no statistics are returned once there is an error of any kind.